docs/site/pages/getting-started.mdx

```diff
@@ -115,8 +115,19 @@ var body: some View {
 
 ```kotlin
 // create Android Player with reference assets plugin
-val player = AndroidPlayer(ReferenceAssetsPlugin())
+val player = AndroidPlayer(
+    listOf(
+        ReferenceAssetsPlugin(),
+        // Any other plugins
+    )
+)
 ```
+Apart from providing a list of plugins while setting up `AndroidPlayer`, you can also provide a `config` object that has the following options:
+ - `debuggable` - Flag to indicate if this is a debug build
+ - `coroutineExceptionHandler` - Custom coroutine exception handler
```

Comment:
this exception handler should be handling all uncaught exceptions while using the runtime coroutine scope